config/config.yaml

```diff
@@ -3,12 +3,14 @@ copy_jobs:
 
 backup_settings:
   backup_worker_max_countdown_in_sec: 0
-  custom_project_list: []
-  projects_to_skip: ['<your-project-id>']
+  custom_project_list: [] #list of projects to backup. If empty, BBQ will backup everything it has read (BigQuery Data Viewer) access to. If list is provided you still need to grant BigQuery Data Viewer role for BBQ service account for each mentioned projects.
```

Comment:
What do you think about describing fields line before? It's hard to read such long lines